templates/nextflow-ecs-task-definition.j2

```diff
@@ -174,10 +174,34 @@ Resources:
           EFSVolumeConfiguration:
             FilesystemId: !Ref EfsFileSystemId
       ContainerDefinitions:
+        - Name: !Sub '${RedisContainerName}-CheckAOF'
```

Comment:
maybe add a comment to indicate that this is a workaround for some jira issue?

Reply:
Good suggestion!